static/scss/partials/onboarding.scss

```diff
@@ -84,6 +85,7 @@
 .c-premium-onboarding-headlines {
     text-align: center;
     margin-bottom: $spacing-xl;
+    line-height: 150% !important;
```

Comment:
Agh, jet lagged coding brain things. Yeah, I was supposed to remove that! Good catch.

Reply:
Just wondering what your best approach is for hard overriding certain attributes? I know we have a `utils` file with a bunch of overriding classes, in the future if we must override for whatever reason do you feel that adding it in that file is good practice?

Reply:
The most important strategy is trying to avoid writing selectors that people will want to override, so that's outside of the scope of this PR, but basically it comes down to avoid reuse of classes unless you're almost 100% certain that the two use cases are exactly the same in every way and will remain so even in future design iterations. Which is rarely the case.

There are tools that can help with this that I'd like us to consider at some point, e.g. [CSS modules](https://css-tricks.com/css-modules-part-1-need/) (not to be confused with [the other CSS modules](https://css-tricks.com/css-modules-the-native-ones/), [Tailwind](https://tailwindcss.com/), and others. I've got a proposal for this that I'm hoping we'll look at in January.

Of course, that doesn't really help us _today_, where we have a lot of (overly, IMHO) generic selectors, some even just styling elements in general, rather than with specific classes. In that case, I look at the selector that's adding styles I don't like, and make sure that mine is more specific. For example, say we have:

```html
<div class="my-class">
  <h2 class="generic-header">
</div>
```

and

```css
/* Somewhere in Protocol */
h2 {
  line-height: 2;
}

/* our_style.css */
.my-class {
  line-height: 1.5;
}
```

Then I can make sure it applies to that h2 by using a more [specific](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) selector:

```css
.my-class h2 {
  line-height: 1.5;
}
```

(Ideally I'd be able to update the original `h2` styling to use a more specific selector that has to be explicitly opted into, but since I don't know every `h2` it's supposed to match with, just using ever more specific selectors is about the best we can do. Which is why people sometimes refer to CSS as an "append-only" language: it's really hard to update/remove existing CSS.)

Reply:
Got it, I've never even heard of CSS Modules, so that was a really interesting read. Thank you! Using specific selectors is definitely super helpful (hence why I really appreciate nesting in SASS).